core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java

```diff
@@ -1,22 +1,44 @@
 package com.scalar.db.storage.dynamo;
 
+import static com.scalar.db.storage.dynamo.DynamoAdmin.waitForTableCreation;
```

Comment:
Let's not use static import for this.

Reply:
Updated in [90037fd]( https://github.com/scalar-labs/scalardb/pull/301/commits/90037fdc8400d01e7387d17c33f4069dc9c02c9a)